website/docs/feature-flag-tutorials/use-cases/ai.md

```diff
@@ -0,0 +1,187 @@
+---
+title: How to use feature flags with AI
+slug: /feature-flag-tutorials/use-cases/ai
+---
+
+Many tech companies today are implementing some form of generative AI with LLMs into their products. This is usually a chatbot or a content generator.
+
+The main way to interact with LLMs today is via a set of APIs, usually either OpenAI, Anthropic or aggregators like Groq. All the APIs usually have similar parameters, like:
+
+-   `model`: The model and the specific version of the model to use.
+-   `prompt`: The prompt to give to the LLM.
+
+Because all these models vary a lot in their capabilities, chances are you'll be testing multiple models, and multiple versions of the same model, and multiple prompts.
+
+This is where feature flags are super useful. And this is what we'll be doing in this tutorial.
+
+We'll be starting with a basic chatbot. I know building a chatbot is a total cliché at this point, but it's a small, self-contained example that everyone knows how to interact with. You can take this example and apply it to any other use case where you need to interact with an LLM via an API.
```

Comment:
We should avoid saying "I" if possible. Also feels a bit too informal to me, how about something like:

Although building a chatbot is a common example, it’s a small and self-contained way to demonstrate these concepts.